examples/app/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@craco/craco": "^6.1.1",
-    "@strapi/design-system": "../../packages/strapi-design-system",
+    "@strapi/design-system": "0.0.1",
```

Comment:
Also, in order to make the dependency resolve, have you used:

```sh
$ npx lerna add @strapi/design-system --scope=strapi-ds-app-example
```

???

Reply:
Nope! I'll update my PR